src/wp-includes/class-wp-term.php

```diff
@@ -195,7 +198,14 @@ public static function get_instance( $term_id, $taxonomy = null ) {
 	 * @param WP_Term|object $term Term object.
 	 */
 	public function __construct( $term ) {
+		$int_fields = array( 'parent', 'term_id', 'count', 'term_group', 'term_taxonomy_id', 'object_id' );
```

Comment:
Could you please add notes for these changes so that others can understand why these changes are needed?

```
@since 6.4.0 Notes for changes
```

Reply:
I see the reason for sanitation due to possibility that data stored in cache can be modified by something else. But if this is the case, sanitization needs to be left where it is.